.vscode/launch.json

```diff
@@ -1,6 +1,23 @@
 {
   "version": "0.2.0",
   "configurations": [
+    {
```

Comment:
This file provides a way to run .NET applications from the root without a devcontainer.